proteobench/io/params/i2masschroq.py

```diff
@@ -73,27 +59,101 @@ def extract_params(fname: pathlib.Path) -> ProteoBenchParameters:
         fragment_mass_tolerance="[-" + _tol_frag + ", " + _tol_frag + "]",
         enzyme=_enzyme,
         allowed_miscleavages=max_cleavage,
-        min_peptide_length=None,  # "spectrum, minimum fragment mz"
-        max_peptide_length=None,  # Not mentioned, up to 38 AA in peptides
+        min_peptide_length=None,  # xtandem: "spectrum, minimum fragment mz"
```

Comment:
So this would be 150, but that is quite indirect. Would you translate it by deviding by the maximum weight on an amino acid?

Reply:
I would not infer. 
For the max, why did you comment "up to 38 AA in peptides"? Where does this information come from? If it is reliable, why not using this? It seems quite short compared to other search engines so it may be interesting to have it parsed.

Reply:
I think it was based on the available example I had.

Reply:
OK, so then do not put anything.
What we could consider is inferring min/max length from the result file. I'll discuss this in https://github.com/orgs/Proteobench/discussions/496